src/headerdef.rs

```diff
@@ -0,0 +1,43 @@
+#[derive(Debug, Display, Clone, PartialEq, Eq, EnumVariantNames)]
```

Comment:
A one-line module-level doc like was done in most other modules is probably nice